doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
```

Comment:
Instead of 'This interface', write it out:  The Websockets interface is a new way ...

Then I would change 'talk' for 'communicate', perhaps ...